app/Http/Controllers/UserController.php

```diff
@@ -116,13 +128,22 @@ public function destroy(User $user)
     {
         try {
             if ($user->id == Auth::user()->id) {
-                return redirect()->back()->with('messageFail', 'Delete failed, Cannot delete myself');
+                return redirect()->back()->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'Delete failed, Cannot delete myself'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 133</br>Each line in an array declaration must end in a comma